config/environments/development.rb

```diff
@@ -1,6 +1,15 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = false
```

Comment:
This should be enabled otherwise it won't work.

Reply:
sure, enabled it to true.